src/containers/App.js

```diff
@@ -9,7 +12,9 @@ class App extends Component {
 
   render() {
     return (
-      <div className="wrapper">
+      <HotKeys
```

Comment:
Yes. As far as I can tell, `react-hotkeys` requires you to wrap the entire React tree that will be listening for hotkeys.